src/encoded/static/components/publication.js

```diff
@@ -45,6 +104,21 @@ const PublicationComponent = (props, reactContext) => {
                 </div>
             : null}
 
+            {context.datasets && context.datasets.length ?
+                <SortTablePanel title="Datasets">
+                    <FetchedData ignoreErrors>
+                        <Param name="datasets" url={datasetsUrl} />
+                        <DatasetsComponent columns={datasetsColumns} />
+                    </FetchedData>
+                </SortTablePanel>
+            : null }
+
+            {context.datasets && context.datasets.length ?
+                <FetchedData ignoreErrors>
+                    <Param name="experiments" url={`/search/?type=Experiment&target.investigated_as!=control`} />
```

Comment:
You can just use double quotes instead of braces and backticks here. ESLint is giving a quotes error here.

Reply:
Removed.